schema/sale_artwork.js

```diff
@@ -32,46 +35,78 @@ const SaleArtworkType = new GraphQLObjectType({
       lot_number: {
         type: GraphQLString,
       },
-      bidder_positions_count: {
-        type: GraphQLInt,
-      },
-      reserve_status: {
-        type: GraphQLInt,
-      },
-      amount_cents: {
-        type: GraphQLInt,
-      },
-      low_estimate_cents: {
-        type: GraphQLInt,
+      currency: {
+        type: GraphQLString,
+        description: 'Currency abbreviation (e.g. "USD")',
       },
-      high_estimate_cents: {
-        type: GraphQLInt,
+      symbol: {
+        type: GraphQLString,
+        description: 'Currency symbol (e.g. "$")',
       },
-      opening_bid_cents: {
-        type: GraphQLInt,
+      reserve_status: {
+        type: GraphQLString,
       },
-      minimum_next_bid_cents: {
-        type: GraphQLInt,
+      is_with_reserve: {
+        type: GraphQLBoolean,
+        resolve: ({ reserve_status }) => reserve_status !== 'no_reserve',
       },
-      currency: {
-        type: GraphQLString,
+      is_bid_on: {
+        type: GraphQLBoolean,
+        resolve: ({ bidder_positions_count }) => bidder_positions_count !== 0,
       },
-      symbol: {
+      reserve_message: {
         type: GraphQLString,
+        resolve: ({ bidder_positions_count, reserve_status }) => {
+          if (reserve_status === 'reserve_met') {
+            return 'Reserve met';
+          } else if (bidder_positions_count === 0 && reserve_status === 'reserve_not_met') {
+            return 'This work has a reserve';
+          }
+          return null;
+        },
       },
+      low_estimate: money({
+        name: 'SaleArtworkLowEstimate',
+        resolve: ({ low_estimate_cents }) => low_estimate_cents,
+      }),
+      high_estimate: money({
+        name: 'SaleArtworkHighEstimate',
+        resolve: ({ high_estimate_cents }) => high_estimate_cents,
+      }),
+      opening_bid: money({
+        name: 'SaleArtworkOpeningBid',
+        resolve: ({ opening_bid_cents }) => opening_bid_cents,
+      }),
+      minimum_next_bid: money({
+        name: 'SaleArtworkMinimumNextBid',
+        resolve: ({ minimum_next_bid_cents }) => minimum_next_bid_cents,
+      }),
+      current_bid: money({
+        name: 'SaleArtworkCurrentBid',
+        resolve: ({ highest_bid_amount_cents, opening_bid_cents }) =>
+          highest_bid_amount_cents || opening_bid_cents,
+      }),
       highest_bid: {
         type: new GraphQLObjectType({
-          name: 'highest_bid',
+          name: 'SaleArtworkHighestBid',
           fields: {
             id: {
               type: GraphQLString,
             },
             created_at: date,
-            amount_cents: {
-              type: GraphQLString,
+            is_cancelled: {
+              type: GraphQLBoolean,
+              resolve: ({ cancelled }) => cancelled,
+            },
+            amount: amount(({ amount_cents }) => amount_cents),
```

Comment:
Nice I like the consistency this is bringing to the sale artwork schema. I'm happy to make the Prediction changes—they're really minimal right now anyways b/c we just started integrated Metaphysics.